### PR TITLE
Remove dataset fields at all

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR removes references to dataset fields at all.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [ ] I have verified that all datasets collect metrics or logs.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/cisco/docs/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The `asa` dataset collects the Cisco firewall logs.
 
 | Field | Description | Type |
 |---|---|---|
-| @timestamp | Date/time when the event originated. This is the date/time extracted from the event, typically representing when the event was generated by the source. If the event source has no original timestamp, this value is typically populated by the first time the event was received by the pipeline. Required field for all events. example: '2016-05-23T08:05:34.853Z' | date |
+| @timestamp | Event timestamp. | date |
```
Comment (ruflin) [inline packages/cisco/docs/README.md:22]: I wonder how this sneaked in?

File: packages/redis/docs/README.md [hunk 2]
```diff
@@ -381,7 +372,7 @@ An example event for `info` looks as following:
 | redis.info.stats.sync.partial.err | The number of denied partial resync requests | long |
 | redis.info.stats.sync.partial.ok | The number of accepted partial resync requests | long |
 | service.address | Client address | keyword |
-| service.version | Version of the service the data was collected from | keyword |
+| service.version | Version of the service the data was collected from. This allows to look at a data set only for a specific version of a service. | keyword |
```
Comment (ruflin) [inline packages/redis/docs/README.md:375]: One more of these new changes 🤔

File: packages/cisco/docs/README.md (hunk 1 above)
Comment (mtojek) [reply packages/cisco/docs/README.md:22]: Not sure and wouldn't like to do the investigation at the moment. Maybe a different field ordering?

File: packages/redis/docs/README.md (hunk 2 above)
Comment (mtojek) [reply packages/redis/docs/README.md:375]: Same here, but I like this longer description :)

File: packages/cisco/docs/README.md (hunk 1 above)
Comment (ruflin) [reply packages/cisco/docs/README.md:22]: Probably, lets ignore it.

**Pull request merged**